docs/docs/howto/coordinator-ha.md

```diff
@@ -0,0 +1,28 @@
+# Coordinator High-Availability
+
+This guide shows how to scale the Contrast Coordinator to more than one instance and make it highly available.
+
+## Prerequisites
+
+Coordinator HA is only available in the enterprise edition of Contrast.
+<!-- TODO(burgerdev): link to documentation of enterprise edition -->
+
+## Workflow
+
+Deploy the Coordinator according to the [basic workflow](../deployment.md).
+By default, there is only one Coordinator instance.
+
+First, we increase the number of instances to 3:
```

Comment:
weird, if it says first but there is only a single step

Reply:
Yeah, seems just as if I hadn't finished writing the docs, after all :sweat_smile: